src/reservation.js

```diff
@@ -0,0 +1,89 @@
+/** @format */
+
+const express = require('express');
+const reservationRouter = express.Router();
+const knex = require('../database');
+
+// reservationRouter.get('/', async (request, response) => {
+//   try {
+//     // knex syntax for selecting things. Look up the documentation for knex for further info
+//     const titles = await knex('reservation').select('title');
+//     response.json(titles);
+//   } catch (error) {
+//     throw error;
+//   }
+//});
+
+//Returns all reservations
+reservationRouter.get('/',async(req,res)=>{
+    try{
+        const allReservations= await knex.select('*').from('Reservation'); 
+        res.json(allReservations); 
+
+    }catch (error) {
```

Comment:
Remember to format your code before committing. It's a very good habit to get used to when working in teams. You could also set VS code to format automatically on save.